src/styles/polygons/polygons_fragment.glsl

```diff
@@ -35,9 +40,14 @@ void main (void) {
     vec4 color = v_color;
     vec3 normal = TANGRAM_NORMAL;
 
-    // Normal material texture (fragment lighting only)
-    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
-        calculateNormal(normal);
```

Comment:
What is expected to happen now when a material is applied as a normal map to regular geometry? 

According to this it seems that regular normal maps (not raster normal maps but user defined uv/planar/triplanar normal maps) are not supported anymore.

What about:
```c
// First, get raster normal if any
#ifdef TANGRAM_RASTER_TEXTURE_NORMAL
    normal = normalize(sampleRaster(0).rgb * 2. - 1.);
#endif

// Second, alter normal with normal map texture if any
#if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
    calculateNormal(normal);
#endif
```

Or, raster normals/material normal maps can be mutually exclusive:
```c
#ifdef TANGRAM_RASTER_TEXTURE_NORMAL
    normal = normalize(sampleRaster(0).rgb * 2. - 1.);
#else
    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
        calculateNormal(normal);
    #endif
#endif
```